### PR TITLE
Pilot updates to exercise and instructor notes

### PR DESCRIPTION
**Pull request by sstevens2**
Updates from issue #95 .  This only includes the first episode I taught. I will put in separate PRs for the other episodes I taught and changed.

I also updated these in the template CodiMD and will also update that before this gets merged with any additional changes.

### CODE REVIEW

**Comment by sstevens2** [comment]
@tobyhodges general question about these notes.  I added a few bulleted lists as notes in the CodiMD that summarize some of the info in the paragraphs.  Do you think we should include those notes in the lesson somewhere or are they okay living separately in the CodiMD template?  In this PR I put them in the instructor notes but there are more in the next episode.

**Comment by tobyhodges** [comment]
It's an interesting question, @sstevens2. I guess it is relatively straightforward to generate a lot of the template CodiMD from the fenced divs for objectives, questions, key points, and exercises. 

Perhaps @zkamvar might consider adding some functionality to `sandpaper` that would allow us to mark up other sections of episode content to be extracted for inclusion in a CodiMD notes template?

File: episodes/15-narrative.md [hunk 1]
```diff
@@ -96,6 +96,27 @@ The [CARE Principles for Indigenous Data Governance](https://doi.org/10.5334/dsj
...
 
+::::::::::::::::::::::::::::::::: instructor 
+
+Outline covering the above paragraph - also used in codiMD template.
+
+### Dataset considerations
+
+- Ethical use (see prompts below)
+- License - CC0 Recommended
+- Complexity - Is it easy to understand? Is it sufficiuently authentic?
```
Comment (tobyhodges) [inline episodes/15-narrative.md:107]:
```suggestion
- Complexity - Is it easy to understand? Is it sufficiently authentic?
```

File: episodes/15-narrative.md [hunk 2]
```diff
@@ -149,6 +170,12 @@ Looking back at one of the exercises you designed before:
 what examples could you include in your narrative to teach learners the skills
 they will need to apply to complete the formative assessments you have designed?
 
+Examples:  
+In the Software Carpentry Plotting and Programming with Python Lesson:
+Exercise to load and inspect americas csv -> In the lesson, load the oceania data table and explore the values with a few different functions.
+In the Python Interactive Data Visualization Lesson in the Incubator:
+Exercise to find the correct widget (a slider widget) for an action and modify the script to use it -> In the lesson the instructor introduces the cheatsheet and documentation and then creates a dropdown widget for the application.
```
Comment (tobyhodges) [inline episodes/15-narrative.md:177]:
```suggestion
Exercise to find the correct widget (a slider) for an action and modify the script to use it -> 
In the lesson the instructor introduces a cheatsheet and documentation for interactive widgets and 
then creates a dropdown widget for the application. 
The slider widget required in the exercise has not been demonstrated but
the preceding example shows all of the necessary steps to add a widget, 
and provides the supporting information that learners can consult to discover
how to implement the specific tool.
```

File: episodes/15-narrative.md [hunk 3]
```diff
@@ -149,6 +170,12 @@ Looking back at one of the exercises you designed before:
 what examples could you include in your narrative to teach learners the skills
 they will need to apply to complete the formative assessments you have designed?
 
+Examples:  
+In the Software Carpentry Plotting and Programming with Python Lesson:
+Exercise to load and inspect americas csv -> In the lesson, load the oceania data table and explore the values with a few different functions.
```
Comment (tobyhodges) [inline episodes/15-narrative.md:175]:
```suggestion
Exercise to load and inspect CSV file for Americas -> 
In the lesson, the Instructor demonstrates how to load the data table for another continent (Oceania)
and explores the values with a few different functions.
This shows learners how to call the function to load the CSV into a data frame,
and demonstrates what success looks like for this task.
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
One typo fix, and a couple of suggestions to make the examples in the exercise more meaningful as guidance for trainees.

**Comment by zkamvar** [comment]
> It's an interesting question, @sstevens2. I guess it is relatively straightforward to generate a lot of the template CodiMD from the fenced divs for objectives, questions, key points, and exercises.
> 
> Perhaps @zkamvar might consider adding some functionality to `sandpaper` that would allow us to mark up other sections of episode content to be extracted for inclusion in a CodiMD notes template?

Make a PR for it and it will be on my radar. The initial wireframes actually do have room for buttons to export a lesson outline, so that could be something that we could wrap into this feature you want.

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**